src/Aspire.Temporal.Server/Aspire.Temporal.Server.csproj

```diff
@@ -1,20 +1,20 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net8.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <IsPackable>true</IsPackable>
     <EnablePackageValidation>true</EnablePackageValidation>
     <PackageId>Asspire.Temporal.Server</PackageId>
```

Comment:
This is intentional because `Aspire.*` prefix is locked.
https://discord.com/channels/333727978460676096/1187220812474896536/1209291102444589106

Reply:
Why not use something like InfinityFlow.Aspire.*

Reply:
done